test/e2e/user_management_test.go

```diff
@@ -314,10 +345,19 @@ func (s *userManagementTestSuite) TestUserDeactivation() {
 			murMember1.Status.ProvisionedTime.String())
 
 		// The user should be set to deactivating, but not deactivated
-		_, err = hostAwait.WaitForUserSignup(userSignupMember1.Name, wait.UntilUserSignupHasConditions(
+		userSignupMember1, err = hostAwait.WaitForUserSignup(userSignupMember1.Name, wait.UntilUserSignupHasConditions(
 			ConditionSet(Default(), ApprovedByAdmin(), Deactivating())...))
 		require.NoError(s.T(), err)
 
+		// Verify that the deactivating notification was created
+		notifications, err := hostAwait.WaitForNotifications(userSignupMember1.Name, toolchainv1alpha1.NotificationTypeDeactivating, 1)
```

Comment:
IIRC, we create notifications using the "compliant username" and not usersignup name.